src/www/ui/api/documentation/openapi.yaml

```diff
@@ -3125,6 +3238,99 @@ components:
           type: string
           example:
             "Main group"
+    TreeViewModel:
+      type: object
+      properties:
+        fileDetails:
```

Comment:
Would suggest it to call `itemDetails` (or something similar) to indicate the `id` here can be used as the `itemId` for other endpoints.